### PR TITLE
Add ci for next

### PR DESCRIPTION
**Pull request by FabioRosado**
This PR will add a specific CI for next now in the right branch 😅   I considered just changing the `TestAndBuild` we have but maybe this one is okay?

@WebReflection  If this works, should we just run the next tests in CI and skip core and all others?

### CODE REVIEW

**Comment by WebReflection** [comment]
To be honest, as we discussed yesterday, I am not planning to change anything in `pyscriptjs` folder but `pyscript.core` `npm run build` should never fail. I will need/want to have current `pyscriptjs` tests in core but for the time being I've just realized that a few things might never work with the integration in pyscriptjs I am trying to do, so I am closing the other PR but I still want CI to test not just pre-commit hooks, thanks!

File: .github/workflows/test-next.yml [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+            MINICONDA_PYTHON_VERSION: py38
+            MINICONDA_VERSION: 4.11.0
+        steps:
+            - name: Checkout
+              uses: actions/checkout@v3
+
+            - name: Install node
+              uses: actions/setup-node@v3
+              with:
+                  node-version: 18.x
```
Comment (WebReflection) [inline .github/workflows/test-next.yml:39]: can we please move to 20.x or the import-map loader won't work?

Comment (FabioRosado) [reply .github/workflows/test-next.yml:39]: Sounds good will do, I'm also removing all the things for pyscriptjs like you suggested 😄

Comment (WebReflection) [reply .github/workflows/test-next.yml:39]: oops ... sorry, not here, in the pyscript.core folder instead, thanks!

**Comment by FabioRosado** [comment]
Woohoo CI is passing 😄 
Also daaammm nice coverage!

File: .github/workflows/test-next.yml [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+            - name: build next
+              working-directory: pyscript.core
+              run: npm run build
+
+            - name: Run next tests
+              working-directory: pyscript.core
+              run: npm run test
+
+            # TODO: DO we want to upload next yet?
```
Comment (WebReflection) [inline .github/workflows/test-next.yml:59]: if we do, it surely won't come from pyscriptjs/build/ articts ;-)

**Review by WebReflection — APPROVED** [review]

**Comment by WebReflection** [comment]
thanks for doing this, very appreciated and needed!

> Also daaammm nice coverage!

it's my bar for any project I write but here I'm overly cheating for all the parts that actually need integration tests ... still I'd like to keep that coverage at that level, and cover what's needed to be covered differently at the integration test level.

**Pull request merged**